fastlane/Fastfile

```diff
@@ -390,6 +391,40 @@ ENV["HAS_ALPHA_VERSION"]="true"
     "#{build_dir}#{name}"
   end
 
+  #####################################################################################
+  # gutenberg_dep_check
+  # -----------------------------------------------------------------------------------
+  # Verifies that Gutenberg is on the last released version 
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane gutenberg_dep_check
+  #####################################################################################
+  desc "Verifies that Gutenberg is on the last released version"
+  lane :gutenberg_dep_check do | options |
+    # Pull the latest develop
+    sh("git checkout develop")
+    sh("git pull")
+    sh("git submodule init")
+    sh("git submodule update")
```

Comment:
`recursive` would also update the `libs/gutenberg-mobile/gutenberg` submodule (you could call it a sub-sub-module :)) it's not required here since we're only checking the tag of `libs/gutenberg-mobile`.

Reply:
👍So I'll merge this as it is and we'll iterate on it if it won't work as expected.

Reply:
👍 Thank you for adding that @loremattei